### ISSUE TITLE
Upgrade Extensions bundle to 3.3.0

### ISSUE THREAD

**Issue by miqm** [Azure/azure-functions-docker#580]
Hello,

Could you upgrade included Function Extensions bundle to [3.3.x](https://github.com/Azure/azure-functions-extension-bundles/releases/tag/3.3.0)? Currently the 3.0.0 version is included in images which even is not mentioned on the MS Docs page: https://docs.microsoft.com/en-us/azure/azure-functions/functions-bindings-register#extension-bundles

**Comment by VpOfEngineering** [comment]
Yes, this will be included in the next host release which will be 3.5.0.

**Comment by miqm** [comment]
Thanks. Any ETA when it will be? days? weeks? months? ....years? :)

**Comment by VpOfEngineering** [comment]
Looking at early next year as we're hitting a code freeze soon.

**Comment by vivekjilla** [comment]
The extension bundle has since been updated in a later host release, and extension bundles now update independently of the images (the desired bundle version can be configured in host.json). Closing as resolved.
